spec/controllers/assignment_invitations_controller_spec.rb

```diff
@@ -44,17 +44,11 @@
 
     after(:each) do
       AssignmentRepo.destroy_all
-      RepoAccess.destroy_all
     end
 
     it 'redeems the users invitation' do
       patch :accept_invitation, id: invitation.key
-
-      assert_requested :post, github_url("/organizations/#{organization.github_id}/teams")
-      assert_requested :post, github_url("/organizations/#{organization.github_id}/repos")
-
       expect(assignment.assignment_repos.count).to eql(1)
```

Comment:
I think so, it verifies that not only we successfully created the `AssignmentRepo` we created it for the right user.